packages/ado-extension/task.json

```
@@ -50,6 +50,7 @@
             "type": "string",
             "label": "Hosting mode: base URL",
             "required": false,
```

Should we set this value to `true`?

```suggestion
            "required": true,
```